_posts/0000-01-02-RestingKnowItAll.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
Sometimes it is **not** easy to learn new things, but it is *rewarding* when you do. I also like [wookieepedia](https://starwars.fandom.com/wiki/Main_Page)!